Source/Extensions/Blazorise.SignaturePad/wwwroot/signaturepad.js

```diff
@@ -0,0 +1,153 @@
+import { getRequiredElement } from "../Blazorise/utilities.js?v=1.2.3.0";
+import "./vendors/sigpad.js?v=1.2.3.0";
+
+const _instances = [];
+
+export function initialize(dotNetAdapter, element, elementId, options) {
+    element = getRequiredElement(element, elementId);
+
+    if (!element)
+        return;
+
+    const sigpad = new SignaturePad(element, {
+        minWidth: options.minLineWidth || 0.5,
+        maxWidth: options.maxLineWidth || 2.5,
+        backgroundColor: options.backgroundColor || "rgba(0,0,0,0)",
+        penColor: options.penColor || "black",
+        velocityFilterWeight: options.velocityFilterWeight || 0.7,
+        dotSize: options.dotSize,
+        throttle: options.throttle || 16,
+        minPointDistance: options.minPointDistance || 0.5,
+    });
+
+    if (options.value) {
+        sigpad.fromData(options.value, { clear: false });
+    }
+
+    if (options.readOnly === true) {
+        sigpad.off();
+    }
+
+    const instance = {
+        options: options,
+        sigpad: sigpad,
+    };
+
+    registerToEvents(dotNetAdapter, instance);
+
+    _instances[elementId] = instance;
+}
+
+export function destroy(element, elementId) {
+    const instances = _instances || {};
+    const instance = instances[elementId];
+
+    if (instance) {
```

Comment:
don't we need to destroy the dotnetobjref? Honestly don't remember.

Reply:
We don't.